client/login/magic-login/style.scss

```diff
@@ -22,6 +24,34 @@
 	}
 }
 
+.layout.is-wpcom-magic-login {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	.magic-login__form-header {
+		margin-bottom: 0;
+
+		@include break-mobile {
+			font-size: $font-title-large;
+		}
+	}
+
+	.logged-out-form {
+		padding-top: 0;
+		.magic-login__form-sub-header {
+			text-align: center;
+		}
+
+		.form-label {
+			font-weight: 400;
+		}
+	}
+
+	.magic-login__form-action {
+		margin-top: 16px;
+	}
+}
+
```

Comment:
Any css for spacing/structure/sizes is applied here where it is more generic. We want structure to be the same regardless of color. 

Conversely, color css is applied specifically under the is-white-login class, since colors will vary based on the color of the login page.